controllers/suite_test.go

```diff
@@ -45,6 +47,8 @@ func TestMain(m *testing.M) {
 func setup() {
 	utilruntime.Must(infrav1.AddToScheme(scheme.Scheme))
 	utilruntime.Must(clusterv1.AddToScheme(scheme.Scheme))
+	utilruntime.Must(expclusterv1.AddToScheme(scheme))
```

Comment:
Yep, it's this. This is actually a compile time failure for me if I check this out.

Reply:
I have it as `scheme.Scheme` and i get the `meta.NoKindMatchError` error. I'm stumped